### PR TITLE
MIT License => Apache 2.0 Migration

### PR DESCRIPTION
**Pull request by brunobowden**
See #1546 for instructions

Closes #1474

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,45 +1,226 @@
- https://github.com/WorldHealthOrganization/app
+https://github.com/WorldHealthOrganization/app
```
Comment (advayDev1) [inline LICENSE:1]: This initial space is required for processing by the Flutter license lister.

**Comment by brunobowden** [comment]
Strange policy check error for "Link with profanity-check" repeated on two runs. Given that this is a ubiquitous line from the Apache 2.0 license. I'm surprised it would cause an error.
https://github.com/WorldHealthOrganization/app/pull/1474/checks?check_run_id=1039189449#step:5:203
```
❌ L91 of LICENSE:

      publicly display, publicly perform, sublicense, and distribute the
```

**Comment by advayDev1** [comment]
Just as FYI for the future - at this time, my personal consent (not speaking on behalf of the project) to relicense my original contributions under this license will depend on (a) WHO's written confirmation of the new license terms, (b) consent from the remaining copyright owners, and (c) legal advice confirming that this change is appropriate.

**Comment by brunobowden** [comment]
Replaced by: https://github.com/WorldHealthOrganization/app/pull/1626

**Pull request closed without merging**